sbatches/farmshare/singularity-run.sbatch

```diff
@@ -17,4 +17,6 @@ fi
 cd $NOTEBOOK_DIR
 
 export SINGULARITY_CACHEDIR=/farmshare/user_data/${USERNAME}/.singularity
```

Comment:
Looks like we forgot to change a $USERNAME here.
```suggestion
export SINGULARITY_CACHEDIR=/farmshare/user_data/${USER}/.singularity
```

Reply:
Ah sorry - I havent touched the other scripts from the original fork from directory. This was one of them. I wasn't sure what this script was for - and why this couldn't be run from the login node.

Reply:
Ah! Well if you aren’t using the scripts and haven’t tested them, it would be ok with me to delete.

Reply:
Sounds good! I will delete every script except for singularity-jupyter and singularity-jupyterlab. I can add more example tested scripts later as it is the case for sherlock.